src/messages.js

```diff
@@ -1,263 +1,262 @@
-      PasswordWarning: "kripya apna password dale"
+      Hibernate: 'seetanidra',
+      Suspend: 'nilambit',
+      Restart: 'punah aarambh',
+      Shutdown: 'bandh',
+      Lock: 'Thala',
+      UserPlaceholder: 'upayogakarta',
+      PasswordPlaceholder: 'paaran shabd',
+      UsernameWarning: 'Kripya apna username dale',
+      PasswordWarning: 'kripya apna password dale'
     }
   },
   lt: {
     LoginScreen: {
-      Hibernate: "Pristabdyti",
-      Suspend: "Nutraukti",
-      Restart: "Restartuoti",
-      Shutdown: "Išjungti",
-      Lock: "Užrakinti",
-      UserPlaceholder: "Vartotojas",
-      PasswordPlaceholder: "Slaptažodis",
-      UsernameWarning: "Įveskite vartotojo vardą.",
-      PasswordWarning: "Įveskite slaptažodį."
+      Hibernate: 'Pristabdyti',
+      Suspend: 'Nutraukti',
+      Restart: 'Restartuoti',
+      Shutdown: 'Išjungti',
+      Lock: 'Užrakinti',
+      UserPlaceholder: 'Vartotojas',
+      PasswordPlaceholder: 'Slaptažodis',
+      UsernameWarning: 'Įveskite vartotojo vardą.',
+      PasswordWarning: 'Įveskite slaptažodį.'
     }
   },
   pl: {
     LoginScreen: {
-      Hibernate: "Uspij",
-      Suspend: "Wstrzymaj",
-      Restart: "Uruchom ponownie",
-      Shutdown: "Wylacz",
-      Lock: "Blokada",
-      UserPlaceholder: "Uzytkownik",
-      PasswordPlaceholder: "Haslo",
-      UsernameWarning: "Wprowadz nazwe uzytkownika.",
-      PasswordWarning: "Wprowadz haslo uzytkownika."
+      Hibernate: 'Uspij',
+      Suspend: 'Wstrzymaj',
+      Restart: 'Uruchom ponownie',
+      Shutdown: 'Wylacz',
+      Lock: 'Blokada',
+      UserPlaceholder: 'Uzytkownik',
+      PasswordPlaceholder: 'Haslo',
+      UsernameWarning: 'Wprowadz nazwe uzytkownika.',
+      PasswordWarning: 'Wprowadz haslo uzytkownika.'
     }
   },
   tr: {
     LoginScreen: {
-      Hibernate: "Hazırda Beklet",
-      Suspend: "Uyut",
-      Restart: "Yeniden Başlat",
-      Shutdown: "Kapat",
-      Lock: "Kilitle",
-      UserPlaceholder: "Kullanıcı",
-      PasswordPlaceholder: "Parola",
-      UsernameWarning: "Lütfen kullanıcı adını gir.",
-      PasswordWarning: "Lütfen parolanı gir."
+      Hibernate: 'Hazırda Beklet',
+      Suspend: 'Uyut',
+      Restart: 'Yeniden Başlat',
+      Shutdown: 'Kapat',
+      Lock: 'Kilitle',
+      UserPlaceholder: 'Kullanıcı',
+      PasswordPlaceholder: 'Parola',
+      UsernameWarning: 'Lütfen kullanıcı adını gir.',
+      PasswordWarning: 'Lütfen parolanı gir.'
     }
   },
   uk: {
     LoginScreen: {
-      Hibernate: "Сплячий режим",
-      Suspend: "Згорнути",
-      Restart: "Перезавантажити",
-      Shutdown: "Вимкнути",
-      Lock: "Заблокувати",
+      Hibernate: 'Сплячий режим',
+      Suspend: 'Згорнути',
+      Restart: 'Перезавантажити',
+      Shutdown: 'Вимкнути',
+      Lock: 'Заблокувати',
       UserPlaceholder: "Ім'я користувача",
-      PasswordPlaceholder: "Пароль",
+      PasswordPlaceholder: 'Пароль',
       UsernameWarning: "Будь ласка, введіть ім'я користувача.",
```

Comment:
Maybe prettier skipped this line? o.O

Reply:
That's weird, I'll double check

Reply:
Actually, it's meant to be that way because there's an apostrophe inside the string